src/connectionbenchmark/benchmarktaskdownload.cpp

```diff
@@ -55,43 +55,60 @@ void BenchmarkTaskDownload::handleState(BenchmarkTask::State state) {
   }
 }
 
+void BenchmarkTaskDownload::connectNetworkRequest(NetworkRequest* request) {
+  logger.debug() << "Connect network requests";
+
+  connect(request, &NetworkRequest::requestUpdated, this,
+          &BenchmarkTaskDownload::downloadProgressed);
+  connect(request, &NetworkRequest::requestFailed, this,
+          &BenchmarkTaskDownload::downloadReady);
+  connect(request, &NetworkRequest::requestCompleted, this,
+          [&](const QByteArray& data) {
+            downloadReady(QNetworkReply::NoError, data);
+          });
+
+  logger.debug() << "Starting request:" << (void*)request;
```

Comment:
Since we're making changes to this file, could you remove the logging of the `(void*)request` pointer?